tests/e2e/smoke/common/common.go

```diff
@@ -220,13 +228,19 @@ func (m *OCRv2State) CreateJobs() {
 	m.SetAllAdapterResponsesToTheSameValue(5)
 	err := m.MockServer.SetValuePath("/juels", 1)
 	Expect(err).ShouldNot(HaveOccurred())
-	bi, err := common.CreateBridges(m.Nodes, m.MockServer)
+	ocr2Contracts := []string{}
+	for _, contract := range m.Contracts {
+		ocr2Contracts = append(ocr2Contracts, contract.OCR2.Address())
+	}
+	err = common.CreateTerraChainAndNode(m.Nodes)
+	Expect(err).ShouldNot(HaveOccurred())
```

Comment:
nit: switch the order here
```suggestion
        err = common.CreateTerraChainAndNode(m.Nodes)
	Expect(err).ShouldNot(HaveOccurred())

	ocr2Contracts := []string{}
	for _, contract := range m.Contracts {
		ocr2Contracts = append(ocr2Contracts, contract.OCR2.Address())
	}
```